solidity/test/bridge/Bridge.Governance.test.ts

```diff
@@ -0,0 +1,4342 @@
+      })
+    })
+  })
+
+  describe("finalizeMovedFundsSweepTimeoutNotifierRewardMultiplierUpdate", () => {
+    context("when the caller is not the owner", () => {
+      it("should revert", async () => {
+        await expect(
+          bridgeGovernance
+            .connect(thirdParty)
+            .finalizeMovedFundsSweepTimeoutNotifierRewardMultiplierUpdate()
+        ).to.be.revertedWith("Ownable: caller is not the owner")
+      })
+    })
+
+    context("when the update process is not initialized", () => {
+      it("should revert", async () => {
+        await expect(
+          bridgeGovernance
+            .connect(governance)
+            .finalizeMovedFundsSweepTimeoutNotifierRewardMultiplierUpdate()
+        ).to.be.revertedWith("Change not initiated")
+      })
+    })
+
+    context(
+      "when the moved funds sweep timeout notifier reward multiplier has not passed",
+      () => {
+        before(async () => {
+          await createSnapshot()
+
+          await bridgeGovernance
+            .connect(governance)
+            .beginMovedFundsSweepTimeoutNotifierRewardMultiplierUpdate(42)
+
+          await helpers.time.increaseTime(constants.governanceDelay - 60) // -1min
+        })
+
+        after(async () => {
+          await restoreSnapshot()
+        })
+
+        it("should revert", async () => {
+          await expect(
+            bridgeGovernance
+              .connect(governance)
+              .finalizeMovedFundsSweepTimeoutNotifierRewardMultiplierUpdate()
+          ).to.be.revertedWith("Governance delay has not elapsed")
+        })
+      }
+    )
+
+    context(
+      "when the update process is initialized and governance delay passed",
+      () => {
+        let tx: ContractTransaction
+
+        before(async () => {
+          await createSnapshot()
+
+          await bridgeGovernance
+            .connect(governance)
+            .beginMovedFundsSweepTimeoutNotifierRewardMultiplierUpdate(42)
+
+          await helpers.time.increaseTime(constants.governanceDelay)
+
+          tx = await bridgeGovernance
+            .connect(governance)
+            .finalizeMovedFundsSweepTimeoutNotifierRewardMultiplierUpdate()
+        })
+
+        after(async () => {
+          await restoreSnapshot()
+        })
+
+        it("should update the moved funds sweep timeout notifier reward multiplier", async () => {
+          const { movedFundsSweepTimeoutNotifierRewardMultiplier } =
+            await bridge.movingFundsParameters()
+          expect(movedFundsSweepTimeoutNotifierRewardMultiplier).to.be.equal(42)
+        })
+
+        it("should emitMovedFundsSweepTimeoutNotifierRewardMultiplierUpdated event", async () => {
+          await expect(tx)
+            .to.emit(
+              bridgeGovernance,
+              "MovedFundsSweepTimeoutNotifierRewardMultiplierUpdated"
+            )
+            .withArgs(42)
+        })
+      }
+    )
+  })
+
+  // continue here with wallet params
```

Comment:
This comment is probably unnecessary.

Reply:
[caa12fa](https://github.com/keep-network/tbtc-v2/pull/329/commits/caa12fa98518fff3ea0aad3a420e624f65647e00)